### PR TITLE
AuthJS quickstart

### PR DESCRIPTION
**Pull request by jakubvul-okta**
## Description:
- Added Auth SDK Quickstart
- Renamed externally-sourced Auth Reference page
- Created redirects for old Auth Reference page
- Changed names of SIW docs to make them consistent

### Resolves:

* [OKTA-129756](https://oktainc.atlassian.net/browse/OKTA-129756)

### CODE REVIEW

File: _source/_code/javascript/okta_auth_sdk.md [hunk 1]
```diff
@@ -1,629 +1,239 @@
...
-If you want to create a token via [OpenID Connect](/docs/api/resources/oidc.html) or [social authentication](/docs/api/resources/social_authentication.html), it's possible to manage the flow with this SDK. To do so, instantiate your client with a `clientId` and `redirectUri`. If you don't have either of those, [set up OpenID Connect with these instructions](https://github.com/oktadeveloper/okta-oauth-spa-authjs-osw/blob/master/Okta-OIDC_SPA_JS-OSW_DevSetupGuide.pdf).
+To initialize the SDK, create a new instance of the `OktaAuth` object:
 
-~~~ javascript
+~~~ js
 var authClient = new OktaAuth({
-  url: 'https://your-org.okta.com',
-  clientId: 'your-client-id',
-  redirectUri: 'https://your.redirect.uri/redirect' // defaults to current url if omitted
+  url: 'https://dev-123456.oktapreview.com',
```
Comment (lboyette-okta) [inline _source/_code/javascript/okta_auth_sdk.md]: we should change this to be their org

File: _source/_code/javascript/okta_auth_sdk.md [hunk 2]
```diff
@@ -1,629 +1,239 @@
...
+    } else if (location.hash) {
+      authClient.token.parseFromUrl()
+      .then(idToken => {
+        console.log(`hi ${idToken.claims.email}!`);
+        // Store parsed token in Token Manager
+        authClient.tokenManager.add('idToken', idToken);
+        console.log(idToken);
+      });
+    } else {
+ // You're not logged in, you need a sessionToken
```
Comment (lboyette-okta) [inline _source/_code/javascript/okta_auth_sdk.md]: let's add some spaces here

File: _source/_code/javascript/okta_auth_sdk.md [hunk 3]
```diff
@@ -1,629 +1,239 @@
...
+      return authClient.signIn({username, password})
+      .then(res => {
+        if (res.status === 'SUCCESS') {
+          authClient.token.getWithRedirect({
+            sessionToken: res.sessionToken,
+            responseType: 'id_token'
+          });
+        }
+      });
+
```
Comment (lboyette-okta) [inline _source/_code/javascript/okta_auth_sdk.md]: let's remove this extra line

File: _source/_code/javascript/okta_auth_sdk.md (hunk 1 above)
Comment (jakubvul-okta) [inline _source/_code/javascript/okta_auth_sdk.md]: Right below this I wrote: 

`Replace each of these property values with ones from your Okta org and application.`

File: _source/_code/javascript/okta_auth_sdk.md (hunk 2 above)
Comment (jakubvul-okta) [inline _source/_code/javascript/okta_auth_sdk.md]: Fixed.

File: _source/_code/javascript/okta_auth_sdk.md (hunk 3 above)
Comment (jakubvul-okta) [inline _source/_code/javascript/okta_auth_sdk.md]: Fixed.

File: _source/_code/javascript/okta_auth_sdk.md (hunk 1 above)
Comment (jmelberg-okta) [inline _source/_code/javascript/okta_auth_sdk.md]: Suggestion: Replace `dev-123456` -> `{yourOrg}`
Same with other values -> `{clientId}`

File: _source/_code/javascript/okta_auth_sdk.md [hunk 4]
```diff
@@ -1,629 +1,240 @@
...
-  sessionToken: 'testSessionToken' // optional if the user has an existing Okta session
-})
-.then(function(tokenOrTokens) {
-  // Manage token or tokens
-})
-.catch(function(err) {
-  // Handle OAuthError
-});
+~~~ js
+console.log(`hi ${idToken.claims.email}!`);
```
Comment (jmelberg-okta) [inline _source/_code/javascript/okta_auth_sdk.md]: nit: `hi` -> `Welcome` or something a little more friendly 😄

File: _source/_code/javascript/okta_auth_sdk.md [hunk 5]
```diff
@@ -1,629 +1,240 @@
...
+      .then(idToken => {
+        console.log(`hi ${idToken.claims.email}!`);
+        // Store parsed token in Token Manager
+        authClient.tokenManager.add('idToken', idToken);
+        console.log(idToken);
+      });
+    } else {
+      // You're not logged in, you need a sessionToken
+      var username = prompt('What is your username?');
+      var password = prompt('What is your password?');
```
Comment (jmelberg-okta) [inline _source/_code/javascript/okta_auth_sdk.md]: Will this be in plaintext or rendered as `******`?

Comment (jakubvul-okta) [inline _source/_code/javascript/okta_auth_sdk.md]: Plaintext.

Comment (jmelberg-okta) [inline _source/_code/javascript/okta_auth_sdk.md]: @lboyette-okta @jakubvul-okta Is this ok? Are there vulnerabilities to using `prompt()`?

**Comment by jmelberg-okta** [comment]
FYI: We will need to update the tests to account for changes in the AuthJS page.
`tests/selenium/spec/check-imported-docs-spec.js`

File: _source/_code/javascript/okta_auth_sdk.md (hunk 5 above)
Comment (jakubvul-okta) [inline _source/_code/javascript/okta_auth_sdk.md]: It's just a quickstart demo...none of this is intended to be used in production.

Comment (mraible) [inline _source/_code/javascript/okta_auth_sdk.md]: Not intended for production, but if it's in the documentation, developers will use it! ;)

Comment (jakubvul-okta) [inline _source/_code/javascript/okta_auth_sdk.md]: That's why there's no UI!

@lboyette-okta do you have a more "realistic" idea for this?

File: _source/_code/javascript/okta_auth_sdk.md (hunk 4 above)
Comment (jakubvul-okta) [inline _source/_code/javascript/okta_auth_sdk.md]: ...I thought Canadians are supposed to be the overly polite ones?

**Pull request merged**